yb-voyager/cmd/import.go

```diff
@@ -156,6 +156,22 @@ func registerCommonImportFlags(cmd *cobra.Command) {
 			"(Note: this is a interim flag until the issues related to 'yb_disable_transactional_writes' session variable are fixed. Refer: https://github.com/yugabyte/yugabyte-db/issues/12464)")
 }
 
+func registerImportDataFlags(cmd *cobra.Command) {
+	cmd.Flags().BoolVar(&disablePb, "disable-pb", false,
+		"true - to disable progress bar during data import (default false)")
+	cmd.Flags().StringVar(&target.ExcludeTableList, "exclude-table-list", "",
+		"List of tables to exclude while importing data (no-op if --table-list is used) (Note: works only for import data command)")
+	cmd.Flags().StringVar(&target.TableList, "table-list", "",
+		"List of tables to include while importing data (Note: works only for import data command)")
+}
+
+func registerImportSchemaFlags(cmd *cobra.Command) {
+	cmd.Flags().StringVar(&target.ImportObjects, "object-list", "",
+		"List of schema object types to include while importing schema (Note: works only for import schema command)")
```

Comment:
In the help text, list valid "object" names and any case-sensitivity constraint if any.

Reply:
It will be our first dynamic help message! Our supported "object" types vary with each source DB type

Reply:
Don't worry about making it "source db type" specific. Just list the type keywords. We already have a check to ensure that an object type is appropriate according to selected source db type.